builder/bundles.go

```diff
@@ -212,18 +194,21 @@ func addUpdateBundleSpecialFiles(b *Builder, bundle *bundle) {
 
 func resolveFilesForBundle(bundle *bundle, packagerCmd []string) error {
 	bundle.Files = make(map[string]bool)
-	for p := range bundle.DirectPackages {
-		queryString := merge(packagerCmd, "repoquery", "-l", "--quiet", p)
-		outBuf, err := helpers.RunCommandOutput(queryString[0], queryString[1:]...)
-		if err != nil {
-			return err
-		}
-		for _, f := range strings.Split(outBuf.String(), "\n") {
-			if len(f) > 0 {
-				addFileAndPath(bundle.Files, resolveFileName(f))
-			}
+
+	queryString := merge(packagerCmd, "repoquery", "-l", "--quiet")
+	for p := range bundle.AllPackages {
+		queryString = append(queryString, p)
+	}
+	outBuf, err := helpers.RunCommandOutput(queryString[0], queryString[1:]...)
+	if err != nil {
+		return err
+	}
+	for _, f := range strings.Split(outBuf.String(), "\n") {
```

Comment:
It would be interesting to see the profile for this, The idea of generate a long string, then break it apart into an array of slices, and eventually discard it all, rather than pulling apart the returned string a bit at a time.